cron/monthly/host-fees.js

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env node
+import '../../server/env';
+
+import { v4 as uuid } from 'uuid';
+
+import { sumTransactions } from '../../server/lib/hostlib';
+import models, { Op } from '../../server/models';
+
+// Only run on the first of the month
+const today = new Date();
+if (process.env.NODE_ENV === 'production' && today.getDate() !== 1 && !process.env.OFFCYCLE) {
+  console.log('NODE_ENV is production and today is not the first of month, script aborted!');
+  process.exit();
+}
+
+const d = process.env.START_DATE ? new Date(process.env.START_DATE) : new Date();
+const rd = new Date(d.getFullYear(), d.getMonth() - 1);
+
+const year = rd.getFullYear();
+const month = rd.getMonth();
+
+const date = new Date();
+date.setFullYear(year);
+date.setMonth(month);
+
+const startDate = new Date(date.getFullYear(), date.getMonth(), 1);
+const endDate = new Date(date.getFullYear(), date.getMonth() + 1, 1);
+
+const dateRange = {
+  createdAt: { [Op.gte]: startDate, [Op.lt]: endDate },
+};
+
+async function main() {
+  const hostsWhere = {
+    isHostAccount: true,
+    isActive: true,
+    HostCollectiveId: { [Op.ne]: null },
```

Comment:
I'm lacking context here, what is a host that has `HostCollectiveId`?

Reply:
We now have this special case of hosts managing their budget. https://github.com/opencollective/opencollective/issues/3161 They work almost like Collectives in that regard. For compatibility reason and to limit the complexity of this feature, they're "hosting themselves". `HostCollectiveId = id`